import/translation_maps/format_map.properties

```diff
@@ -57,3 +57,4 @@ VideoDisc           = DVD
 VideoOnline         = Video
 VideoReel           = Video
 Video               = Video
+Website             = Website
```

Comment:
It would probably be good to add an end-of-file line break to all of the properties files just for consistency.